src/components/PeoplePage.tsx

```diff
@@ -1,33 +1,68 @@
+/* eslint-disable max-len */
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
 
 export const PeoplePage = () => {
-  return (
-    <>
-      <h1 className="title">People Page</h1>
-
-      <div className="block">
-        <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
 
-          <div className="column">
-            <div className="box table-container">
-              <Loader />
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        const peopleFromServer = await getPeople();
 
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+        setPeople(peopleFromServer);
+      } catch (error) {
+        setErrorMessage('Something went wrong');
```

Comment:
Consider logging the error object to the console for debugging purposes. This can help in identifying the specific error that occurred during the fetch operation.